File: odds.js
/* 
given a range which begins at l and terminates 
at r return all odd numbers within the range
*/

function oddNumbers(l, r) {
    let arr = []
    for (i = l; i >= l && i <= r; i++) {
        if (i % 2 !== 0) {
            arr.push(i)
        };
    };
    return arr
};

console.log(oddNumbers(3, 14))

File: odds.py
# ODDS #

# Given two integers, l and r, return all the odd
# numbers between l and r (l and r inclusive).
# The function must return an array of numbers.

def find_the_odds(l, r):
    odds = list(filter(lambda x: x % 2 != 0, range(l, r)))
    return odds


print(find_the_odds(3, 14))
